### PR TITLE
Update for 2023.1

### PR DESCRIPTION
**Pull request by markgoddard**
- Update Kolla Ansible services inventory template for Antelope
- Elasticsearch -> OpenSearch
- Update branch used to 2023.1
- Use Rocky 9 image
- Use Ark instead of Test Pulp

### CODE REVIEW

**Review by MaxBed4d — APPROVED** [review]
With the removal of the `stackhpc_release_pulp_url: "http://pulp-server.internal.sms-cloud:8080/"` does there need to be a variable to point at `ARK`? Or was the inclusion of that variable in order to bypass automatically using `ARK`? If this is the case, then I am happy with the changes made!

**Comment by markgoddard** [comment]
> With the removal of the `stackhpc_release_pulp_url: "http://pulp-server.internal.sms-cloud:8080/"` does there need to be a variable to point at `ARK`? Or was the inclusion of that variable in order to bypass automatically using `ARK`? If this is the case, then I am happy with the changes made!

The default points at Ark, defined in `etc/kayobe/pulp.yml`.

File: deploy.sh [hunk 1]
```diff
@@ -7,13 +7,13 @@ set -eu
 
 # These parameters must be changed to your own fork 
 KAYOBE_CONFIG_REPO="https://github.com/example/stackhpc-kayobe-config"
-KAYOBE_CONFIG_BRANCH=stackhpc/yoga
```
Comment (Alex-Welsh) [inline deploy.sh:10]: Possibly should have kept this change to the antelope branch but oh well. I don't think we'll come back to Yoga

Comment (markgoddard) [reply deploy.sh:10]: hmm, I don't think any of this is valid for yoga except the change to `stackhpc_release_pulp_url`. I'll revert.

**Pull request merged**